scripts/run_OQStandard.sh

```diff
@@ -2,9 +2,13 @@
 # ==========================================================================
 # Script for running scenario calculations in the National Canada Risk Model
 # ==========================================================================
+
+set -exo pipefail
```

Comment:
If I recall correctly, the motivation of setting `set -o pipefail` etc. is another consequence of me changing `&>` to `2>&1 | tee`, as the subsequent pipe in `| tee` always succeed and wouldn't stop the script even if `oq engine` failed unless `set -o pipefail` etc. is set.